src/components/30-organisms/table/index.js

```diff
@@ -0,0 +1,39 @@
+import { LitElement, css, unsafeCSS } from 'lit-element';
+import defineOnce from '../../../utils/define-once';
+import tableCss from './index.scss';
+
+// subcomponents that can only be used with axa-table:
+
+// semantics are done with aria: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Table_Role
+class AXATable extends LitElement {
+  static tagName = 'axa-table';
+
+  static get tagName() {
+    return 'axa-table';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(tableCss)}
+    `;
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    const table = this.querySelector('table');
+    if (table) {
```

Comment:
What if the table is not there, is there even a use case for that?

Reply:
defensive programming.. if table is not here, nothing happens

Reply:
I would take the lit element firstUpdated() method for the connectedCallback, because there lifecycle callback really called once "Implement firstUpdated to perform one-time work after the element’s template has been created." @source lit element docs

Reply:
Not sure, but if the table is not there, then we have a huge problem, right? Should we really check that? Why don't we check it in all the other methods?